inc/composer/class-command.php

```diff
@@ -175,6 +200,21 @@ protected function get_env() : array {
 	protected function start( InputInterface $input, OutputInterface $output ) {
 		$output->writeln( '<info>Starting...</>' );
 
+		if ( $input->hasParameterOption( '--docker-sync' ) || $this->has_docker_sync_file() ) {
```

Comment:
Hmm I'm not sure if we want `|| $this->has_docker_sync_file()`. If I want to start without docker-sync, after I've tried it, how would I do that?

I'd recommend we read from the config for the value if you don't want someone to have to pass `--docker-sync` every time. (also, support for a local config file would help there!)

Reply:
The files for both mutagen and docker-sync are removed after a destroy so right now you'd have to either destroy and re-up or remove the file manually. I could make it so the flag accepts a value like `--docker-sync=off` to remove it.

This is needed because the `docker-compose.yml` file has to be generated every time to apply settings. Once you start the server in this way it needs to stay consistent when running other commands so that the differences can be factored in eg. when restarting, stopping or destroying the server.